auth-api/tests/unit/api/test_task.py

```diff
@@ -53,13 +55,49 @@ def test_fetch_tasks_no_content(client, jwt, session):  # pylint:disable=unused-
     assert rv.status_code == http_status.HTTP_200_OK
 
 
-def test_fetch_tasks_with_status(client, jwt, session):  # pylint:disable=unused-argument
+@pytest.mark.parametrize('test_name, endpoint', [
+    ('with status', 'status=OPEN'),
+    ('with relationship_status', 'relationshipStatus=PENDING_STAFF_REVIEW'),
+    ('with type', 'type=BCeID Admin'),
+    ('with name', 'name=hgffdh')
+])
+def test_fetch_tasks_with_params(test_name, client, jwt, endpoint, session):  # pylint:disable=unused-argument
+    """Assert that the tasks can be fetched."""
+    user = factory_user_model()
+    factory_task_service(user.id)
+
+    headers = factory_auth_header(jwt=jwt, claims=TestJwtClaims.staff_role)
+
+    if (endpoint == 'type=BCeID Admin'):
+        pdb.set_trace()
```

Comment:
Hmm, what is pdb.set_trace() for? Usually you can just set a breakpoint in code and VSCODE should stop.. if you're having some troubles getting that going.. let me know 

apparently `breakpoint()` works too